docs/contributions/online-contributions.md

```diff
@@ -100,10 +98,10 @@ This is covered in detail in  [Memberships](/membership/online-membership-sign-u
 
 ### Profile tab
 
-If you want to collect information from contributors beyond the essential fields
-required to make a contribution, such as age, interests and skills,
-you can include existing CiviCRM Profiles at the beginning or end of a
-contribution page. You can also create new profiles.
+If you want to collect information such as age, interests and skills that goes
+beyond the essential fields required to make a contribution, you can include
+existing [CiviCRM Profiles](/organising-your-data/profiles) at the beginning or
```

Comment:
@agh1 can you put a .md on the end of the url as its an internal url

Reply:
None of the other URLs on the page have it--I thought I was told once *not* to do it.  If the proper style is to include the `.md` I'm happy to fix all the links in the file.

Reply:
I know. I'm sorry this is so confusing! It's explained here: https://docs.civicrm.org/dev/en/latest/documentation/markdown/#internal-url-standards 

The `.md` isn't actually necessary, but adding it helps us avoid broken internal links. For links between guides, the URL should be treated as external and so you should grab the URL you see in the browser, which won't have a `.md`.